### DISCUSSION TITLE
Configuración para desplegar en Github Pages

### DISCUSSION THREAD

**Discussion by AngelCruzO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#484] [Q&A]
Disculpen las molestias, de casualidad alguien tiene la configuración para desplegar proyectos de Vue a Github Pages

**Comment by doguedogue** [comment]
No lo he probado, vue solo he publicado en netlify

Pero checa esto a ver si te sirve:
https://learnvue.co/2020/09/how-to-deploy-your-vue-app-to-github-pages/#step-1-set-publicpath-in-vue-config-js-to-our-repository-name

**Comment by doguedogue** [comment]
También probé con este manual, pero no pude, se me queda en blanco, etc. 😭
https://platzi.com/tutoriales/1111-vuejs/3441-como-hacer-deploy-de-una-aplicacion-de-vuejs-hacia-github-pages-usando-vue-cli-3/

**NOTA**: de hecho es más fácil copiar la carpeta de distribución para publicar que hacerlo automático 🚀

Igual es más fácil subirlo a Azure Static Apps o netlify, con esos si los probé 😁

Tu si pudiste con github pages? 🤯

**Reply by AngelCruzO** [reply]
No me ha quedado, ayer subí la dist directa a un repo y se quedó en blanco. Yo creo que lo voy a subir a netlify. 

Ya luego experimento, porque con el archivo deploy, se necesita configurar el webpack y el día que lo probé trone mi proyecto jaja
